types/node/process.d.ts

```diff
@@ -1,8 +1,121 @@
 declare module "process" {
-    import * as net from "node:net";
-    import * as os from "node:os";
     import * as tty from "node:tty";
     import { Worker } from "node:worker_threads";
+
+    interface BuiltInModule {
+        "assert": typeof import("assert");
```

Comment:
Hopefully we remember to add new modules here? Seems easy to forget, but I don't see a better way, really.

Reply:
The same...